lib/livebook_web/controllers/auth_html/index.html.heex

```diff
@@ -62,45 +62,9 @@
   </div>
 
   <div
-    :if={@app_sessions != [] or @empty_apps_path?}
+    :if={@any_public_app? or @empty_apps_path?}
     class="w-full h-full px-4 py-8 flex justify-center items-center"
   >
-    <div class="w-full flex flex-col items-center">
-      <div class="text-gray-700 text-xl font-medium">
-        Public apps
-      </div>
-      <div :if={@app_sessions != []} class="mt-5 max-w-[400px] w-full flex flex-col space-y-4">
-        <.link
-          :for={session <- @app_sessions}
-          navigate={~p"/apps/#{session.app_info.slug}"}
-          class="px-4 py-3 border border-gray-200 rounded-xl text-gray-800 pointer hover:bg-gray-50 flex justify-between"
-        >
-          <span class="font-semibold"><%= session.notebook_name %></span>
-          <.remix_icon icon="arrow-right-line" class="" />
-        </.link>
-      </div>
-      <div :if={@empty_apps_path?} class="mt-5 text-gray-600">
-        <div>
-          No app notebooks found. <br />Follow these steps to list your apps here:
-        </div>
-        <ol class="mt-4 pl-4 flex flex-col space-y-1 list-decimal list-inside">
-          <li>
-            Open a notebook
-          </li>
-          <li>
-            Click <.remix_icon icon="rocket-line" class="align-sub text-lg" />
-            in the sidebar and configure the app as public
-          </li>
-          <li>
-            Save the notebook to the
-            <span class="font-medium"><%= Livebook.Config.apps_path() %></span>
-            folder
-          </li>
-          <li>
-            Relaunch your Livebook app
-          </li>
-        </ol>
-      </div>
-    </div>
+    <%= live_render(@conn, LivebookWeb.AuthAppListLive, container: {:div, class: "w-full"}) %>
```

Comment:
What if we only invoke this if there are apps? This way we don’t establish socket connections unless necessary in the home page?

Reply:
`:if={@any_public_app? or @empty_apps_path?}` we do :D

Reply:
Unless you mean we moving `@empty_apps_path?` here and rendering just the list?